Makefile

```diff
@@ -150,22 +150,22 @@ operator-sdk:
 	fi
 
 .PHONY: generate-csv
-generate-csv: operator-sdk dist-csv-generator
+generate-csv: operator-sdk kustomize dist-csv-generator
```

Comment:
please expand a bit the commit message to explain why this commit is needed

Reply:
will do, it is used in hack/generate-csv tool

Reply:
done